core/src/main/java/com/vml/es/aem/acm/core/code/arg/NumberArgument.java

```diff
@@ -0,0 +1,40 @@
+package com.vml.es.aem.acm.core.code.arg;
+
+import com.vml.es.aem.acm.core.code.Argument;
+import com.vml.es.aem.acm.core.code.ArgumentType;
+
+public class NumberArgument extends Argument<Number> {
+    private Number min;
+
+    private Number max;
+
+    private Number step;
+
+    public NumberArgument(String name) {
+        super(name, ArgumentType.SLIDER);
```

Comment:
number -> slider?

Reply:
also now I am confused, we have integerNumber decimalNumber and... number? ... end users could also be confused